### ISSUE TITLE
Allow key upload without positive test

### ISSUE THREAD

**Issue by HeiDasGri** [corona-warn-app/cwa-wishlist#126]
Warning of other users would be much faster, if a warning could be issued before a test is done and analysed. I like to illustrate this request on two hypothetical examples:
- I have been identified by an health authority, that I am a primary contact of an infected person and have some unspecific symptoms. Here it is not necessary to wait but to handle quickly. Thus I get a pin from authority and could upload my contacts instantly. Due to quarantine, no further contacts
- I get a warning by the corona warn app. This automatically give me the possibility to upload my contacts

In both cases it is useful to update the upload after positive test and to revoke the upload after negative test. 

It would be perfect if not only the keys are published, but when also the status "infected" or "potential infected" could be aligned with. This might result in a yellow warning "You have had contact to a person which fears to have infected. Ceep calm, avoid contacts, and wait, if the status will update in a view days"

Of curse this scenario has to be discussed with RKI. And before starting this task, the existing process needs to speed up first.
corona-warn-app/cwa-backlog#2

**Comment by cfritzsche** [comment]
You can check https://github.com/corona-warn-app/cwa-wishlist/issues/24 for a request containing what you ask and related discussion.

**Comment by HeiDasGri** [comment]
@cfritzsche : Thanks for hint. My second example much better match to the story there. The first example is not about secondary contacts but about shorten the time, Nevertheless it is the same general idea so I close the item here and discus both there

**Comment by cfritzsche** [comment]
You are right. The title of that issues has always been about nth degree contacts. But in the end the goal is to shorten the time and a notification for symptoms without test was also discussed there as an option.

**Comment by Ulenrich** [comment]
1. Any abuse of a second degree warning would weaken this CWA system. An authorized key for the second degree warning therefore is very much needed. 2. Another issue with this: What about a second degree warning of a person who never is tested?
Therefore:
Why not supply every testing kit with its authorization key at the start. Thus the medical testing person can give this key to the person tested, who then can upload an authorization key that not yet is in a status positively tested?
And: Such an authorized second degree key upload automatically gets purged again after a defined period if this key not is uploaded a second time as positively tested ....

**Comment by HeiDasGri** [comment]
@Ulenrich A good summary of the necessary steps. Exactly a thing, which RKI should consider. Please take a look to 
[Issue 24](https://github.com/corona-warn-app/cwa-wishlist/issues/24) where a similar idea is discussed in more detail. Both cases, getting a test kit and get a app warning should be used in a positive way to inform further people.
